package.json

```diff
@@ -600,11 +600,14 @@
                             },
                             "logfilePath": {
                                 "type": "string",
-                                "description": "A path to a file where all brightscript console output will be written. If null or empty, file logging will be disabled."
+                                "description": "A path to a file where all brightscript console output will be written. If null or empty, file logging will be disabled.",
+                                "scope": "resource"
```

Comment:
`"scope": "resource"` doesn't belong in debug configuration settings. Remove this entry.

Reply:
done, thanks

Reply:
removed thanks